RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -811,6 +811,34 @@ string freeRunCombatString(monster enemy, location loc, boolean inCombat)
 	string pre_banish = freeRunCombatStringPreBanish(enemy, loc, inCombat);
 	if (pre_banish != "") return pre_banish;
 
+	//Standard free-runs
+	if (!inAftercore() && have_effect($effect[Everything Looks Green]) == 0)
+	{
+		if(auto_haveSpringShoes())
```

Comment:
Also use auto is valid for spring away. Similar reasoning to comment below

Reply:
Done